modules/3d/src/ptcloud/sac_segmentation.hpp

```diff
@@ -0,0 +1,147 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_3D_SAC_SEGMENTATION_HPP
+#define OPENCV_3D_SAC_SEGMENTATION_HPP
+
+#include "opencv2/3d/ptcloud.hpp"
+
+namespace cv {
+
+//! @addtogroup _3d
+//! @{
```

Comment:
`src/` code is not a part of OpenCV documentation. These doxygen markers could be removed.

Reply:
fixed.